src/pages/LenderProfile/LenderProfile.vue

```diff
@@ -95,18 +103,25 @@ export default {
 			publicId: '',
 			allAchievements: [],
 			lenderIsPublic: false,
+			enableBadgeContent: false,
+			badgeImage: null,
+			badgeCategory: '',
 		};
 	},
 	apollo: {
 		preFetch(config, client, { route }) {
 			const currentRoute = route.value ?? route ?? {};
 			const publicId = currentRoute.params?.publicId ?? '';
 
-			return client.query({
-				query: lenderPublicProfileQuery,
-				variables: { publicId }
-			});
-		},
+			const utmCampaign = currentRoute?.query?.utm_campaign ?? '';
+			const loadBadgeInfo = utmCampaign.includes('badge_') && utmCampaign.includes('social_share');
+			const badgeKey = utmCampaign.split('badge_')[1];
```

Comment:
Should we be checking if `badgeKey` is defined and acting different below if it's not defined?